apps/site/pages/docs/component-customization/creating-component.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: 'Creating a Component'
+sidebar_label: 'Getting Started'
+---
+
+import { Callout } from 'nextra-theme-docs'
+
+<header>
+
+# Creating a Component
+
+</header>
+
+<Callout type="warning" emoji="⚠️">
+This documentation is currently under development.
+</Callout>
+
+In this guide, you'll learn how to create a component from scratch and use it in your FastStore project. For the purpose of this guide, we'll create a `Price` component to display the price of a product in the `Product Details` component.
+
+|   |   |
+|---|---|
+| ![faststore-component](https://vtexhelp.vtexassets.com/assets/docs/src/faststore-price-component___d9d0e2fbf65e6680d0f26bed7a0e9b8e.png) The **FastStore `Price` component** displays the price of a given product, discount and total values.| ![custom-component](https://vtexhelp.vtexassets.com/assets/docs/src/custom-price-component___3b8a888867b6bf354541236163ef3d79.png) The **custom `Price` component** created can display any information that you set. In this case it renders the name `Price`|
+
+## Step by step
+
+### Step 1: Create the `Price` file
+
+1. Open your FastStore project in a code editor and navigate to `src` folder.
+2. Create a new folder called `components` inside the `src` folder.
```

Comment:
We should make clear that this structure is suggested but NOT required. They can place the component in any subdirectory of the `src` folder.

The only requirement is on the overrides and ProductDetails (in this example) files.

Reply:
Added a callout to inform the suggestion.